README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Please at least add a top section to the project readme explaining what the project is. You can keep the other content here too if you like.